versioned_docs/version-V3/guides/liquidity-mining/overview.md

```diff
@@ -0,0 +1,42 @@
+- `pool`: The address of the Uniswap V3 pool in which liquidity must be provided.
+- `startTime`: The UNIX timestamp at which rewards start to be distributed.
+- `endTime`: The UNIX timestamp at which rewards start to decay.
+- `refundee`: The address which has the right to reclaim any leftover rewards after the `Incentive` has concluded.
+
+## Staking
+
+So, now that we have an idea of what an `Incentive` looks like, let's try understand how a user might participate. Note that this section requires a basic understanding of how Uniswap V3 position NFTs work.
+
+The first action a user must take in order to begin participating in an `Incentive` is to _deposit_ their position NFT into the [canonical staking contract address](https://github.com/Uniswap/uniswap-v3-staker#deployments), effectively temporarily giving custody over their NFT to this contract. This is necessary because, as we'll see later on, the staking contract needs to be able to guarantee that liquidity cannot be removed from NFTs participating in the program.
+
+Once deposited, a user may then _stake_ their deposited token into any number of active `Incentive`s for the Uniswap V3 pool their NFT is tied to. Note that this can happen atomically with an initial _deposit_. Staked NFTs then immediately start to earn rewards, _as long as the underlying positions are in range_. 
+
+## Reward Math
+
+So, how are rewards actually allocated to staked NFTs? Recall that `Incentive` creators are essentially picking _an amount of `rewardToken`s to distribute per second to in-range LPs_, over some interval. Let's call this value the reward rate. Between `startTime` and `endTime`, the reward rate remains constant, and for every second that goes by, tokens are allocated proportionally _among all in-range liquidity at the time_. Crucially, this counts not just _staked_ liquidity, but _all_ liquidity. So, incentive creators should pick a reward rate that they deem worthwhile to distribute across (potentially) all in-range LPs for the duration of the program.
+
+## Program Conclusion
+
+But what happens if some of the liquidity active during an `Incentive` is not staked? Some `rewardToken`s were set aside for this unstaked liquidity, but they cannot be claimed. This is where the `refundee` comes into play. After a program has concluded, the `refundee` can claim any remaining `rewardToken`s in the contract.
+
+There are two conditions that must be met for a program to be considered concluded:
+
+1. `block.timestamp >= endTime`: In other words, the program must have officially ended.
+2. All staked NFTs must be unstaked: After a program ends, _anyone_ may unstake any NFT (claiming their `rewardToken`s for them in the process). This ensures that even if all users do not unstake themselves, someone can unstake them manually so that the `refundee` can claim remaining tokens.
```

Comment:
might be slightly more clear to say "unstake any NFT (triggering a claim of their `rewardToken`s to be sent back to the original owner)".   Current phrase might lead people to believe others can claim their rewards.